bcipy/tasks/session_data.py

```diff
@@ -0,0 +1,184 @@
+"""Module for functionality related to session-related data."""
+from typing import Dict, List
+from collections import Counter
+
+
+class StimSequence:
+    """Represents a sequence of stimuli."""
+
+    def __init__(self,
+                 stimuli: List[str],
+                 eeg_len: int,
+                 timing_sti: List[float],
+                 triggers: List[List],
+                 target_info: List[str],
+                 target_letter: str,
+                 current_text: str,
+                 copy_phrase: str,
```

Comment:
This should be optional in the case of other tasks (free_spell). Perhaps, `target_text`?